app/components/avo/index/resource_controls_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= content_tag :div, class: "space-x-2 flex flex-row justify-between ml-2 has-record-selector:ml-0" do %>
+<%= content_tag :div, class: "#{params[:view_type] == "grid" ? "flex-wrap" : "space-x-2"} flex flex-row justify-evenly ml-2 has-record-selector:ml-0" do %>
```

Comment:
```suggestion
<%= content_tag :div, class: "#{view_type == "grid" ? "flex-wrap" : "space-x-2"} flex flex-row justify-evenly ml-2 has-record-selector:ml-0" do %>
```

Can we please extract `view_type` inside the component?

Reply:
Done! @adrianthedev